easybuild/easyconfigs/n/novoalign/novoalign-3.09.01_fix_hardcoded.patch

```diff
@@ -0,0 +1,42 @@
+# Fix hard-coded R and Perl paths
+# Novermber 29th 2018 by B. Hajgato (Free University Brussels - VUB)
```

Comment:
novope2bed.pl also need harcoded path fixed

Reply:
@akesandgren Indeed, I looked it over. Fixed.